CONTRIBUTING.md

```diff
@@ -1,62 +1,152 @@
 
-  - The `title` of the new SVG.
-  - A `hex` color value that matches the brand or logo's main accent color. (Without the `#` pound symbol.)
-  - The `source` URL of the logo being used.
+We welcome icon requests. However, before you submit a new issue please make sure the icon:
 
-Here is the object for the GitHub logo as an example:
+* Is a _popular_ brand.
+* Has not already been requested.
+* Can be represented in a monochromatic format.
 
-```json
-{
-  "title": "GitHub",
-  "hex": "181717",
-  "source": "https://github.com/logos"
-},
-```
+When submitting a new icon request, please submit the issue with the following information:
+
+* **Title:** Include the brand name. Example: `Request: GitHub Icon`
+* **Body:** Include links to official sources for the brand's icon and official colors (e.g. media kits, brand guidelines, SVG files etc.)
+
+## How To Add An Icon
+
+### 1. Identify Official Logos and Colors
+
+Most of the icons and brand colors on SimpleIcons have been derived from official sources. Using official sources helps ensure that the icons and colors in SimpleIcons accurately match the brand they represent. Thankfully, this is usually a simple process as organizations often provide brand guides and high quality versions of their logo for download.
+
+Official high quality brand logos and brand colors can usually be found in the following locations:
+
+1. About pages, Press pages, Media Kits, and Brand Guidelines.
+1. Website headers
+1. Favicons
+1. Wikimedia (which should provide a source)
+
+Working with an SVG version of the logo is best. In the absence of an SVG version, other vector filetypes may work as well such as EPS. In the absence of vector logos, a vector can be created from a high quality rasterized image, however this is much more labor intensive.
+
+For color, the brand's primary web color should be used. Simple Icons stores brand colors in the standard 6 character hexadecimal format.
+
+### 2. Extract The Icon From The Logo
+
+There are many different tools for editing SVG files, some options include:
+
+| Name | Description | Platform | Price | Wikipedia |
+| :---- | :---- | :----: | :----: | ----: |
+| [Inkscape](https://inkscape.org/en/)| Vector Graphics Editor | Win, Mac, Linux | Free | [Wiki](https://en.wikipedia.org/wiki/Inkscape) |
+| [Adobe Illustrator](https://www.adobe.com/products/illustrator.html) | Vector Graphics Editor | Win, Mac | $ - $$$ | [Wiki](https://en.wikipedia.org/wiki/Adobe_Illustrator) |
+| [IcoMoon](https://icomoon.io/) | Icon Editing/Management Tool | Online | Free | -- |
+
+Using your preferred tool you should:
+
+1. Isolate the icon from any text or extraneous items.
+1. Change the icon's viewbox/canvas/page size to 24x24.
+1. Scaled the icon to fit the viewbox, while preserving the icon's original proportions. This means the icon should be touching at least two sides of the viewbox.
+1. Center the icon horizontally and vertically.
+1. Export the icon as an SVG.
```

Comment:
Interesting numbering scheme 😛

Reply:
Auto-updating numbered lists - super useful!

Reply:
Interesting 🤔 Nice to know!